ansible/roles/ocp4-workload-workshop-admin-storage/tasks/clean-admin.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: prepare - ensure no serviceaccount
+  k8s:
+    state: absent
+    definition:
+      kind: ServiceAccount
+      metadata:
+        name: admin-user
+        namespace: labguide
+
+- name: prepare - ensure no configmap
+  k8s:
+    state: absent
+    definition:
+      kind: configmap
+      metadata:
+        name: admin-env
+        namespace: labguide
+
+- name: prepare - ensure no service
+  k8s:
+    state: absent
+    definition:
+      kind: service
+      metadata:
+        name: admin
+        namespace: labguide
+
+- name: remove the labguide project
+  command: oc delete project labguide
+  ignore_errors: yes
+
+- name: wait for project to be deleted before continuing
+  command: oc get project labguide
+  register: result
+  retries: 30
+  failed_when: result.stderr.find('Error from server (NotFound)') != 0
+  until: result.stderr.find('Error from server (NotFound)') == 0
```

Comment:
I think you can use

```shell
oc delete project labguide --wait=true
```

That way you don't have to "wait for project to be deleted before continuing"

Reply:
@fridim What about using `wait` in `k8s`?

How do I ensure ansible installed is v2.8.*?

Reply:
I had
```
yum: name=ansible-2.8.*
```
but that didn't seem to help on first run on the task right after the "upgrade". Or maybe I needed to add wait?